scripts/config

```diff
@@ -33,10 +36,7 @@ EOF
 }
 
 get__ssh_public_key() {
-    cat << EOF
-ask:
-  en: "**Public key**: \`$(cat /root/.ssh/id_${app}_ed25519.pub || echo '')\`"
-EOF
+    echo $(cat /root/.ssh/id_${app}_ed25519.pub || echo '')
```

Comment:
If I may comment the existing code (not yours), I am not sure why `|| echo ''`. If I do:
```bash
cat /does-not-exist || echo ''
```

The result is:
```
cat: /does-not-exist: No such file or directory
```

I would rather suggest:
```suggestion
    local key_file="/root/.ssh/id_${app}_ed25519.pub"
    [ -r "$key_file" ] && cat "$key_file"
```

Or maybe (if the return code is not checked):
```suggestion
    cat "/root/.ssh/id_${app}_ed25519.pub" 2>/dev/null
```